resources/content/schema/service/service-auth.json

```diff
@@ -2,7 +2,12 @@
     "authorize" : {
 
         "registrationToken.accountId" : "cr",
-
+        "physicalHost": "crud",
```

Comment:
Why do we need to allow service accounts full crud access to physical hosts? Also, below you've done physicalHost.*, I believe we been avoiding full blown wildcards like that in our auth files.